### ISSUE TITLE
The description of an Option[io.circe.Json] does not work.

### ISSUE THREAD

**Issue by pme123** [softwaremill/tapir#1378]
I want to use `io.circe.Json` for wrapped values that have no case class representations.

This works for Seq but not for Option.

```
  case class ExampleMessageIn(
                               @description("The name of your example")
                               name: String,
                               @description("My seq that does work")
                               numbers: Seq[Json] = Seq(ch,li),
                               @description("My json is great but optional") // this line does not create a docu.
                               maybeJson: Option[Json] = Some(li),
                               @description("My optional string is working")
                               maybeString: Option[String] = Some("hello")
                             )
```
![image](https://user-images.githubusercontent.com/3437927/125944314-4a92bc90-d0de-4093-acb6-a2913dfd400d.png)

How to reproduce:

Run this code: https://gist.github.com/pme123/87b50f25e8f9ca7a6bccd27ee58d18c1

**Comment by pme123** [comment]
I found another problem with the JSON Object.

If I have 2 Json objects it takes the description of the first:

![image](https://user-images.githubusercontent.com/3437927/126276022-4ac1c6d9-6ab4-4afd-bb88-22245efcca9f.png)

```
@description("The customer with the requested `clientKey`.")
customer: Json = customer,
@description("The consultant of the customer.")
consultant: Json = consultant,
```
Shall I create another ticket for this case?

**Comment by adamw** [comment]
@pme123 This is a known issue, but more with the openapi spec, than with tapir. Should be fixed in openapi 3.1, though, when `$ref`s will be able to have descriptions. See: https://github.com/softwaremill/tapir/issues/1203
